### DISCUSSION TITLE
"Remove Similars" feature like duplicates?

### DISCUSSION THREAD

**Discussion by TommySharpNZ** [ente-io/ente#1585] [Enhancements]
My family routinley will just mash the shutter button on their phone and take multiple photos of the same person or scene.
In terms of "duplicates" they are different with different fielnames, hashes EXIF data etc but visually they will be so close!

I think a great feature would be a way to easily view similar photos to the one I have selected and then be able to cycle through with a side by side comparison that lets me easily keep the best shot. So if I find a better shot than where I started then the original is deleted and now I'm comparing photos with my new best shot.

This would also be handy for sorting through exact photos but where I have multiple versions of different file sizes.

I know it's still early days but I'm loving the feature set so far!
I'm still on the quest to find the perfect replacement for the old "Picasa" setup I had years ago :-)

Keep up the great work everyone!

**Comment by vishnukvmd** [comment]
Thanks for the feedback @TommySharpNZ, totally get the use case.

Supporting "culling" so that you can easily pick out the best out of the burst of photos you've clicked is on our radar. Hard to commit to a timeline, but we will definitely ship this.

**Reply by andrewbanchich** [reply]
Similar to this - I'd love "remove similars" not because of bursts, but because Google Photo edits will save the edited versions as separate photos. All the photos I've ever edited are duplicated, usually the only difference being a slight color / contrast / brightness change. I'd love if Ente could detect these automatically and let me choose which ones I want to keep.
